stories/atoms/components/toast/Toast.stories.mdx

```diff
@@ -45,7 +47,7 @@ Several severity levels make it possible to fully configure the type of message
 
 <Story
   name="Overview"
-  argTypes={{ isOpened: { control: false } }}
+  argTypes={{ isOpened: { control: false }, description: { control: false } }}
```

Comment:
It's a disappointment that you can't control that property. Why not force the control as a string to be able to play anyway?

Reply:
Great catch, I didn't know we could do that with storybook